Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -79,27 +84,31 @@ public PostProcessorTests(TestContext testContext)
     [TestMethod]
     public void Constructor_NullArguments_ThrowsArgumentNullException()
     {
-        Invoking(() => new PostProcessor(null, null, null, null, null, null)).Should()
+        Invoking(() => new PostProcessor(null, null, null, null, null, null, null)).Should()
             .Throw<ArgumentNullException>()
             .And.ParamName.Should().Be("sonarScanner");
 
-        Invoking(() => new PostProcessor(scanner, null, null, null, null, null)).Should()
+        Invoking(() => new PostProcessor(scanner, null, null, null, null, null, null)).Should()
+            .Throw<ArgumentNullException>()
+            .And.ParamName.Should().Be("sonarEngine");
```

Comment:
There's an extension for these that could be used everywher in this method
```suggestion
            .WithParameterName("sonarEngine");
```